api/controllers/game/pass.js

```diff
@@ -9,7 +9,7 @@ module.exports = function (req, res) {
       let gameUpdates = {
         turn: game.turn + 1,
         passes: game.passes + 1,
-        log: [ ...game.log, `${player.username} passes` ],
+        log: [ ...game.log, `${player.username} passed` ],
```

Comment:
Standardizing this between game state and legacy API so tests can consistently assert the same log expectations. Might be a good idea to make a pass through all logs to ensure they're the same tense. Picked past tense; we can see how that feels